plugins/imjournal/imjournal.c

```diff
@@ -761,6 +799,10 @@ CODESTARTsetModCnf
 			free(fac);
 		} else if (!strcmp(modpblk.descr[i].name, "usepidfromsystem")) {
 			cs.bUseJnlPID = (int) pvals[i].val.d.n;
+			errmsg.LogError(0, RS_RET_DEPRECATED,
```

Comment:
looking right now, it seems to me that the error message is not moved (and I also think the current place is appropriate, I was just suggesting a different function).